packages/video_player_videohole/README.md

```diff
@@ -0,0 +1,121 @@
+# video_player_videohole
```

Comment:
What does "videohole" in the package name mean? The suffix in package names usually represents the platform name (e.g. `_tizen`) or the name of the backing API (e.g. `_avfoundation`).

Reply:
`videohole` means that this plugin using the way of hole, is different from video player (texture)

Reply:
Hmm. What does "using the way of hole" mean? As far as I understand the code, the video is rendered "in front" of the Flutter widget but not to the back side through a hole. Do you have a link to a public document that explains what "videohole" is?